src/org/exist/dom/INode.java

```diff
@@ -33,7 +33,12 @@
  */
 public interface INode<D extends org.w3c.dom.Document, T extends INode> extends org.w3c.dom.Node,
     INodeHandle<D>, Comparable<T> {
-    
+
+    /**
+     * The node is a <code>Namespace</code>.
+     */
+    public static final short NAMESPACE_NODE              = 13;
```

Comment:
I start this discussion because think that this values are used at storage. If you are sure that it's not used for storage then it ok to go.